flink-runtime/src/main/java/org/apache/flink/runtime/jobgraph/JobVertex.java

```diff
@@ -579,6 +579,28 @@ public interface InitializeOnMasterContext {
         int getExecutionParallelism();
     }
 
+    /** The context exposes some runtime infos for finalization. */
+    public interface FinalizationOnMasterContext {
```

Comment:
FinalizationOnMasterContext -> FinalizeOnMasterContext

To be aligned with `InitializeOnMasterContext`.